.pipelines/templates/publish-helm-chart.yaml

```diff
@@ -1,20 +1,23 @@
 steps: 
   - bash: |
-      az login --identity
-      az acr login --name $(registry.name)
+      az login --identity > /dev/null 2>&1
+      az acr login --name $(REGISTRY_NAME) > /dev/null 2>&1
     displayName: Login to acr
+    env: 
+      REGISTRY_NAME: $(REGISTRY_NAME)
   - bash: |
       if [[ -n "${CHECKOUT_TAG}" ]]; then
         cp $(image.dir)/$(chart.name)-${CHECKOUT_TAG}.tgz .
         echo 'pushing chart'
-        oras push $(staging.registry):${CHECKOUT_TAG} ./$(chart.name)-${CHECKOUT_TAG}.tgz:application/tar+gzip --debug
+        oras push $(STAGING_REGISTRY):${CHECKOUT_TAG} ./$(chart.name)-${CHECKOUT_TAG}.tgz:application/tar+gzip --debug
```

Comment:
`STAGING_REGISTRY` should be `REPOSITORY_NAME` as we are pushing to a specific repo in a specific registry.

Reply:
Good idea